plugins/chat/assets/javascripts/discourse/models/chat-channel.js

```diff
@@ -93,7 +93,7 @@ export default class ChatChannel {
   threadsManager = new ChatThreadsManager(getOwner(this));
   messagesManager = new ChatMessagesManager(getOwner(this));
 
-  @tracked _unreadThreadIds = new TrackedSet();
+  @tracked _unreadThreadOverview = new TrackedMap();
```

Comment:
Do you need a map here? or an object would be enough?

Reply:
I tried to use a `TrackedObject` but it did not trigger updates in the UI when I was replacing the value of it, I'm not sure why. Besides in this case it is just a key/value map (threadId => Date()) so TrackedMap is a good fit, if we need more values in it we could try object again.

Reply:
I can't tell you why without seeing code but it should have worked.

Reply:
IDK, anyway I think the TrackedMap is working fine since we only have one key/value pair